zipline/finance/performance/tracker.py

```diff
@@ -281,7 +278,6 @@ def to_dict(self, emission_type=None):
             _dict.update({'daily_perf': self.todays_performance.to_dict()})
         elif emission_type == 'minute':
             _dict.update({
-                'intraday_risk_metrics': self.intraday_risk_metrics.to_dict(),
                 'minute_perf': self.todays_performance.to_dict(self.saved_dt)
```

Comment:
Not a big deal, but since this and the dict above both contain only a single item, they could be ```_dict['minute_perf'] = ...``` instead of ```.update({...})```.